auto-update.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Keeps wc-api-dev up to date based on GitHub releases.
+ */
+class WC_API_Dev_Updater {
+
+	// Plugin file path / slug
+	protected $file = 'wc-api-dev/wc-api-dev.php';
+	protected $github_response = null;
+
+	/**
+	 * Hooks into the plugin update system.
+	 */
+	public function __construct() {
+		add_filter( 'pre_set_site_transient_update_plugins', array( $this, 'modify_transient' ), 10, 1 );
+		add_filter( 'plugins_api', array( $this, 'plugin_popup' ), 10, 3 );
+		add_filter( 'auto_update_plugin', array( $this, 'auto_update' ), 10, 2 );
+		add_filter( 'upgrader_post_install', array( $this, 'after_install' ), 10, 3 );
+	}
+
+	/**
+	 * Adds the wc-api-dev plugin to the list of allowed plugins for auto update.
+	 */
+	function auto_update( $update, $item ) {
+		if ( in_array( $item->slug, array( 'wc-api-dev' ) ) ) {
```

Comment:
Why not use a simple string comparison here?

`if ( 'wc-api-dev' === $item->slug ) {`